tls/README.md

```diff
@@ -0,0 +1,48 @@
+# Agent Check: TLS
+
+## Overview
+
+This check monitors [TLS][1] protocol versions, certificate expiration & validity, etc.
+
+## Setup
+
+### Installation
+
+The TLS check is included in the [Datadog Agent][2] package.
```

Comment:
`Datadog Agent` should be linked to:
https://app.datadoghq.com/account/settings#agent

Right not it is linked to the yaml file.